administrator/components/com_languages/src/Controller/InstalledController.php

```diff
@@ -92,8 +92,8 @@ public function switchAdminLanguage()
 			$file = JPATH_ADMINISTRATOR . '/language/' . $cid . '/' . $cid . '.xml';
 		}
 
-		$info         = Installer::parseXMLInstallFile($file);
-		$languageName = $info['name'];
+		$info = LanguageHelper::parseXMLLanguageFile($file);
```

Comment:
```suggestion
		$info         = LanguageHelper::parseXMLLanguageFile($file);
```
Align "=" with the one in the row below.

Reply:
Aligned the "=" as suggested.